src/literals/literals.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Debug, PartialEq, Eq)]
+pub enum Literal {
+    String(String),
```

Comment:
We've got 
- string - "hello"
- char - \`c\`
- array - {1, 5, 3}
- int - 20 - b1010 - 0x0a

Reply:
Arrays are defined by a more complex grammar, thus not found by the lexer. Integers should be classified as IntegerLiteral and we can support different radixes as you mentioned